docs/reference/migration/migrate_7_13.asciidoc

```diff
@@ -117,13 +117,62 @@ aggregations are rarely useful and often unintended.
 [%collapsible]
 ====
 *Details* +
-Support for multiple paths in the `path.data` setting is now deprecated. We
-introduced this option as a way to support multi-disk setups. It has since been
-a source of user complaints due to confusing and unintuitive behavior.
+The `path.data` setting accepts a list of data paths, but if you specify
+multiple paths then the behaviour is unintuitive and usually does not give the
+desired outcomes. Support for multiple data paths is now deprecated and will be
+removed in 8.0.0.
 
 *Impact* +
-Specify a single path in `path.data`. To use multiple disks, use a RAID
-hardware configuration or similar hardware solution.
+Specify a single path in `path.data`. If needed, you can create a filesystem
+which spans multiple disks with a hardware virtualisation layer such as RAID,
+or a software virtualisation layer such as Logical Volume Manager (LVM) on
+Linux or Storage Spaces on Windows. If you wish to use multiple data paths on a
+single machine then you should run one node for each data path.
+
+If you currently use multiple data paths in a
+<<high-availability-cluster-design,highly available cluster>> then you can
+migrate to a setup that uses a single path for each node without downtime using
+a process similar to a <<restart-cluster-rolling,rolling restart>>: shut each
+node down in turn and replace it with one or more nodes each configured to use
+a single data path. At each step this process will discard the data spread
```

Comment:
I think we should clarify whether:

1. Re-using one of the data paths for a node is OK or if we require to delete the data path to have it recreated. I tend to think it might work?
2. Re-using all the data paths for new single-data-path nodes. I believe this cannot work and should be warned against.

Reply:
I suspect it would actually work to keep one of the data paths around, but I don't think we should recommend that. It definitely wouldn't work to re-use more than one, you'd end up with multiple nodes with the same ID.